lib/routes/bahamut/creation-index.ts

```diff
@@ -1,4 +1,5 @@
-const utils = require('./utils');
```

Comment:
Please [use namespace](https://docs.rsshub.app/joinus/new-rss/start-code#creating-namespace) `gamer` https://github.com/DIYgod/RSSHub/blob/5b072a3d55e972d10203ae39cd2ca6808955f632/lib/routes/gamer/namespace.ts#L1
instead of creating variations of the same namespace.